### DISCUSSION TITLE
[FR] Compatible with HomeTab Plugin

### DISCUSSION THREAD

**Discussion by Moyf** [chrisgurney/obsidian-note-toolbar#200] [Ideas]
[Home Tab](https://github.com/olrenso/obsidian-home-tab) is a useful plugin with 37k downloads, it can replace the "New tab" with a browser-like page, allowing users to quickly search notes or open recent/pinned notes.
![image](https://github.com/user-attachments/assets/544ddffa-6ce0-4494-afd3-402a3fd58555)

Is it possible to display NTB's toolbar in Home tab's "New tab"?
It would be great if I can use float button of NTB in this page:
![image](https://github.com/user-attachments/assets/1750796a-55d0-4974-ada2-90efacb4cfda)
(Especially on my mobile App) 🥺

**Comment by chrisgurney** [comment]
@Moyf Thank you for the request. I have the ability to use toolbars in other file types, aside from notes, [on the roadmap](https://github.com/chrisgurney/obsidian-note-toolbar/wiki/Roadmap).

Although I'm not familiar with the HomeTab plugin, the feature I have planned _might_ support it as well. I'm planning on using the `data-type` attribute here: `<div class="workspace-leaf-content" data-type="canvas">`. Does the HomeTab plugin use this attribute?

**Reply by Moyf** [reply]
> @Moyf Thank you for the request. I have the ability to use toolbars in other file types, aside from notes, [on the roadmap](https://github.com/chrisgurney/obsidian-note-toolbar/wiki/Roadmap).
> 
> Although I'm not familiar with the HomeTab plugin, the feature I have planned _might_ support it as well. I'm planning on using the `data-type` attribute here: `<div class="workspace-leaf-content" data-type="canvas">`. Does the HomeTab plugin use this attribute?

Just checked and it seems that HomeTab use `home-tab-view`:
![image](https://github.com/user-attachments/assets/bbfba224-690e-4c48-81e7-db571c8c7487)
`<div class="workspace-leaf-content" data-type="home-tab-view">`

Looks like  it's feasible!  😄

**Reply by chrisgurney** [reply]
@Moyf

> `<div class="workspace-leaf-content" data-type="home-tab-view">`
> 
> Looks like  it's feasible!  😄

Great! Thank you for checking!

**Comment by chrisgurney** [comment]
See also #78 and #130

**Comment by chrisgurney** [comment]
@Moyf Home Tab is now supported in the latest beta. Give it a try and let me know how it works for you.

**Comment by Moyf** [comment]
![image](https://github.com/user-attachments/assets/6b038549-a715-4750-abba-540533e745a3)

Yeah, works great! Thank you  <3
